docs/source/configuration/external_timelock_service_configs/timelock_client_config.rst

```diff
@@ -38,12 +40,53 @@ Required parameters:
            Note that client names must be non-empty and consist of only alphanumeric characters, dashes and
            underscores (succinctly, ``[a-zA-Z0-9_-]+``) and for backwards compatibility cannot be the reserved word ``leader``.
 
+Optional parameters:
+
+.. note::
+
+    Specifying the ``serversList`` (a ``ServerListConfig``) in the install configuration has been deprecated, but is
+    maintained for backward compatibility. Please switch to declaring the ``ServerListConfig`` in the runtime
+    configuration as soon as possible.
+
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *    - Property
+         - Description
+
     *    - serversList::servers
          - A list of all hosts. The hosts must be specified as addresses, i.e. ``https://host:port``.
-           At least one server must be specified. AtlasDB assumes that the Timelock Servers being pointed at
-           are part of the same Timelock cluster.
+           AtlasDB assumes that the Timelock Servers being pointed at are part of the same Timelock cluster.
+           If this is not provided, it defaults to the empty list.
 
-Optional parameters:
+    *    - serversList::sslConfiguration
```

Comment:
Does this change with the move to live reloading?  Can you have live reloaded list of Timelock nodes but a static security block?

Reply:
Yes - but the static security block *must* be provided in the runtime config. I've added another paragraph to the note here explaining how this is handled.